regions_www/manual.html

```diff
@@ -0,0 +1,436 @@
+<p>By experimenting with the ‘Freeze lines’ option and zooming into specific areas of interest, the level of cycling at more local levels can be explored. And by setting the basemap to ‘Satellite’ when zoomed into specific parts of the road network, the space available to create new cycle paths and car-free zones can be explored.</p>
+<p>Overall, there are many ways that the tool can be used to improve cycling provision. It is likely that more features will be added in future versions The table below explains the various options in the interactive map.</p>
+<table class="table table-hover table-responsive">
+<tbody>
+<tr>
+<td style="vertical-align: top; width: 1054px;">
+<p><span style="font-weight: bold; text-decoration: underline;">Scenario:</span></p>
+<ol style="list-style-type: decimal">
+<li>Census 2011 Reported Cycle Commuting</li>
+<li>Government Target: this scenario is based on the UK government’s proposed target (as set out in its draft <a href="https://www.gov.uk/government/uploads/system/uploads/attachment_data/file/364791/141015_Cycling_Delivery_Plan.pdf">Cycling Delivery Plan</a>) to double cycling in England. Taking population increases and trip rate decreases into account, the target implies doubling the proportion of trips made by cycling nationwide. Trips that have a large distances or that already have a high rate of cycling will see cycling increase less than two-fold; trips with a below average current rate of cycling but high potential (based on the distance) will see cycling increase more than two-fold.</li>
+<li>Gender equality: Women have the same cycling rate as men</li>
+<li>Go Dutch: Cycling propensity at the levels seen in the Netherlands</li>
+<li>Electric bicycles: Willingness to cycle longer trips increases with widespread use of electric bicycles</li>
+</ol>
+</td>
+<td style="vertical-align: top; width: 535px;">
+<img src="/home/robin/npct/pct-shiny/www/assets/help/scenario-selection.png" alt="Scenario Selection" />
+</td>
+</tr>
+<tr>
+<td style="vertical-align: top; width: 1054px;">
+<p><span style="font-weight: bold; text-decoration: underline;">Zone Attribute</span></p>
+<ol style="list-style-type: decimal">
+<li>Scenario Level of Cycling is the model predicted rate of cycling in the given area for the chosen scenario</li>
+<li>Scenario Increase in Cycling is the corresponding number of additional cycle commuters over and above Census 2011</li>
+<li>None: No zone attribute is displayed</li>
+</ol>
+</td>
+<td style="vertical-align: top; width: 535px;">
+<img src="/home/robin/npct/pct-shiny/www/assets/help/zone-attribute.png" alt="Zone Attribute" />
```

Comment:
or this one ^.  The links all seem to point to your local directory @Robinlovelace, sadly I've no Rmarkdown expertise so can't really suggest much, hope its an easy fix :smile: